addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -3,9 +3,12 @@ odoo.define('website.editor.snippets.options', function (require) {
 
 var core = require('web.core');
 var Dialog = require('web.Dialog');
-var weWidgets = require('wysiwyg.widgets');
 const wUtils = require('website.utils');
 var options = require('web_editor.snippets.options');
+// TODO (review) better way to do that? this shows `popupOptions` as unused
+//      but is required as we extend a registry Class defined in it..
+var popupOptions = require('website.s_popup_options');
```

Comment:
Just `require` then, do not put the result in a variable. No need of comment either it is self explanatory: you don't need the result but you require it so that this module is called after.